infra/resources/prod/function_profile.tf

```diff
@@ -0,0 +1,230 @@
+}
+
+locals {
+  function_profile = {
+    name = "profile"
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME       = "node"
+      FUNCTIONS_WORKER_PROCESS_COUNT = 8
+      NODE_ENV                       = "production"
+
+      COSMOSDB_NAME              = "db"
+      COSMOSDB_URI               = data.azurerm_cosmosdb_account.cosmos_api.endpoint
+      COSMOSDB_KEY               = data.azurerm_cosmosdb_account.cosmos_api.primary_key
+      COSMOSDB_CONNECTION_STRING = format("AccountEndpoint=%s;AccountKey=%s;", data.azurerm_cosmosdb_account.cosmos_api.endpoint, data.azurerm_cosmosdb_account.cosmos_api.primary_key)
+
+      MESSAGE_CONTAINER_NAME = "message-content"
+      QueueStorageConnection = data.azurerm_storage_account.storage_api.primary_connection_string
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      // TODO: the following 3 env seems to be not used anymore, at least searching it in the codebase show no results
+      AssetsStorageConnection    = "TODO: REMOVE"
+      STATUS_ENDPOINT_URL        = "https://api-app.io.pagopa.it/info"
+      STATUS_REFRESH_INTERVAL_MS = "300000"
+
+      // TODO: Rename to SUBSCRIPTIONSFEEDBYDAY_TABLE_NAME
+      SUBSCRIPTIONS_FEED_TABLE = "SubscriptionsFeedByDay"
+      MAIL_FROM                = "IO - l'app dei servizi pubblici <no-reply@io.italia.it>"
+      DPO_EMAIL_ADDRESS        = "dpo@pagopa.it" //TODO: seems to not be used anymore
+      PUBLIC_API_URL           = "https://api-app.internal.io.pagopa.it/"
+      FUNCTIONS_PUBLIC_URL     = "https://api.io.pagopa.it/public"
+
+      // Push notifications
+      // TODO: the following 3 env seems to be not used anymore, at least searching it in the codebase show no results
+      AZURE_NH_HUB_NAME                       = "io-p-ntf-common"
+      NOTIFICATIONS_QUEUE_NAME                = "push-notifications"
+      NOTIFICATIONS_STORAGE_CONNECTION_STRING = "TODO: REMOVE"
+
+      // Service Preferences Migration Queue
+      MIGRATE_SERVICES_PREFERENCES_PROFILE_QUEUE_NAME = "profile-migrate-services-preferences-from-legacy"
+      FN_APP_STORAGE_CONNECTION_STRING                = data.azurerm_storage_account.storage_app.primary_connection_string
+
+      // Events configs
+      EventsQueueStorageConnection = data.azurerm_storage_account.storage_apievents.primary_connection_string
+      EventsQueueName              = "events" # reference to https://github.com/pagopa/io-infra/blob/12a2f3bffa49dab481990fccc9f2a904004862ec/src/core/storage_apievents.tf#L7
+
+      BETA_USERS = data.azurerm_key_vault_secret.fn_app_beta_users.value // TODO: searching this config in the codebase show no results
```

Comment:
following [this PR](https://github.com/pagopa/io-functions-app/pull/154/files#diff-8f4625a1cf4cb245a2f9e780a9a17ff6a7132b8e070997144a67499d7ff793b4L50), we can safely remove this